### ISSUE TITLE
Ray not working "GTP not working" error on a cloud pc (liquidsky)

### ISSUE THREAD

**Issue by tartaric** [zakki/Ray#121]
Hello,

I tried to run Ray (differents versions, from 4.2.1 to the latest 4.20.4) on a cloud pc (Liquidsky) and it didn't work with the error "GTP not working" while on my real pc it works. I'm using Sabaki software to run the GTP, but I have also tried with Drago and Smartgo, but didn't work either.

So I don't know why it's not working, maybe because the OS is not a classical one (as Window 8 or windows 10) but it's writter "Window server edition 2016". The CPU is a XEON E5 2690 v3 2.6 ghz and the GPU is a NVIDIA GRIDE M60-8Q.

I tried with and without the command --no-gpu but didn't work either.

So what do you think would br the problem?

Thanks for your help!

**Comment by zakki** [comment]
Try "ray" in command prompt without GUI front end like Sabaki.
And after "Init CNTK ok", type "genmove b".

**Comment by tartaric** [comment]
Thanks for your message.

When I tried to launch Ray from the command prompt, it immediately wrote: "Rn has stopped working" as you can see on this screen:

http://hpics.li/e00f6ad

**Comment by zakki** [comment]
Does original Ray work?
http://computer-go-ray.com/eng/index.html
http://www.computer-go-ray.com/dlmagicV1.0/dlmagic.php?url=http://www.computer-go-ray.com/file/ray-8.0-windows.zip

**Comment by tartaric** [comment]
Ray32 is working but when I try running ray64 the same error appears: "**Ray has stopped working**".

On Sabaki, this message appears when I try running any versions: "**connection error**" and "init CNTK ok" doesn't even appear.

I have to add i'm running a Pc on the cloud (thanks to "Paperspace" website and the OS should be Windows 10 but it's written "Windows Server 2016 Datacenter"). **The CPU is a Xeon E5-2623 v4 and GPU is a NVIDIA Quadro P5000.**

Here is a screen from it, I hope it could help you maybe seing something.

![cloud pc](https://user-images.githubusercontent.com/31356777/29795973-0f78fcac-8c4f-11e7-97d8-5fdcb92383bb.png)

I tried to run Leela GPU accelerated version and it worked perfectly (destroying the Zen version on my real computer, which uses a I5 6400 CPU and no GPU by the way…).

**I think a very important and basic file is missing but I don't know which one**. I think so because, previously, I had to download Microsoft Visual C++ 2015 cause Vcomp140.dll and others dll files were missing according to Ray when I tried to run it through "ray.exe" (which is what you call command prompt I guess?).

Thanks for your help again zakki! :)
